app/email/email_test.go

```diff
@@ -40,34 +40,71 @@ func TestSendInviteEmails(t *testing.T) {
 
 	th.UpdateConfig(func(cfg *model.Config) {
 		*cfg.ServiceSettings.EnableEmailInvitations = true
+		*cfg.EmailSettings.SendEmailNotifications = false
 	})
+	t.Run("SendInviteEmails", func(t *testing.T) {
+		emailTo := "test@example.com"
+		mail.DeleteMailBox(emailTo)
 
-	require.NotNil(t, th.BasicUser)
-	require.NotNil(t, th.BasicChannel)
+		err := th.service.SendInviteEmails(th.BasicTeam, "test-user", th.BasicUser.Id, []string{emailTo}, "http://testserver")
+		require.NoError(t, err)
 
-	emailTo := "test@example.com"
-	mail.DeleteMailBox(emailTo)
+		var resultsMailbox mail.JSONMessageHeaderInbucket
+		err2 := mail.RetryInbucket(5, func() error {
+			var err error
+			resultsMailbox, err = mail.GetMailBox(emailTo)
+			return err
+		})
+		if err2 != nil {
+			t.Log(err2)
+			t.Log("No email was received, maybe due load on the server. Skipping this verification")
+		} else if len(resultsMailbox) > 0 {
+			require.Len(t, resultsMailbox, 1)
```

Comment:
I think we can remove the `len(resultsMailbox)` condition check if we opt to use `t.Skip` above as it's already going to stop if there are no resultsMailBox.